### ISSUE TITLE
gataddrinfo() system call error text unstable leading to failed test

### ISSUE THREAD

**Issue by liamdennehy** [phpseclib/phpseclib#1501]
`$errstr` return by `fsockopen()`:
https://github.com/phpseclib/phpseclib/blob/3e9a474a2285d8e6c4453f45df9d0b97dfda87bc/phpseclib/Net/SSH2.php#L1203

... is used in thrown error:
https://github.com/phpseclib/phpseclib/blob/3e9a474a2285d8e6c4453f45df9d0b97dfda87bc/phpseclib/Net/SSH2.php#L1206

... but the value appears to be unstable leading to a failed test:

```
1) SFTPWrongServerTest::testLoginToInvalidServer
Failed asserting that two strings are identical.
--- Expected
+++ Actual
@@ @@
-'Cannot connect to dummy-server:22. Error 0. php_network_getaddresses: getaddrinfo failed: Name or service not known'
+'Cannot connect to dummy-server:22. Error 0. php_network_getaddresses: getaddrinfo failed: No address associated with hostname'

./src/phpseclib/tests/Functional/Net/SFTPWrongServerTest.php:15
```

**Comment by terrafrost** [comment]
Good catch! I'm gonna cherry pick this to the 3.0 branch and then merge to master.

**Comment by terrafrost** [comment]
Cherry picking this one was a little trickier than usual lol. I did `git remote add liam git://github.com/liamdennehy/phpseclib.git` and then `git fetch` but it turns out that that git fetch included a 3.0 tag that conflicted with the 3.0 branch.

Anyway, this has been merged.

Thanks!

**Comment by liamdennehy** [comment]
Yeah my master has a stable version tag to make pulling into another library of mine a bit easier, apologies for the confusion :)
